packaging/flatpak/dev.sh

```
@@ -0,0 +1,4 @@
+./build.sh 0.0.3.2a ../../dist fast
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)